python-sdk/nuscenes/eval/predict/tests/test_metrics.py

```diff
@@ -1,8 +1,28 @@
 import unittest
+from unittest.mock import MagicMock, patch
+from typing import Dict
+import json
+import os
 
 import numpy as np
+from shapely.geometry import shape, MultiPolygon
 
+from nuscenes.predict import PredictHelper
 from nuscenes.eval.predict import metrics
+from nuscenes.eval.predict.data_classes import Prediction
+
+
+def mock_load_drivable_area_polygons(mock_helper: MagicMock) -> Dict[str, MultiPolygon]:
+    del mock_helper
+
+    def load_polygon(filename: str):
+        return shape(json.load(open(os.path.join(DIRECTORY, filename)))).buffer(0)
+
+    DIRECTORY = os.path.dirname(os.path.realpath(__file__))
+    return {'singapore-onenorth': load_polygon('onenorth_drivable_area.json'),
+            'singapore-hollandvillage': load_polygon('holland_village_drivable_area.json'),
+            'singapore-queenstown': load_polygon('queenstown_drivable_area.json'),
+            'boston-seaport': load_polygon('boston_drivable_area.json')}
```

Comment:
Where do these mock polygons come from? Can we just add the code to create them from the official map?